packages/react/src/line-chart.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { LineChart as LC } from "@carbon/charts";
+import {
+  ChartData,
+  LineChartOptions,
+} from "@carbon/charts/interfaces";
+import BaseChart, { TabularData } from "./base-chart";
+
+type LineChartData =
+  | TabularData
+  | ChartData
```

Comment:
we actually need to remove this type or update it to match the tabular format. Not sure why it's still in our types

Reply:
all mentions of `datasets` & `labels` in the charting data should also disappear from the react types or tests

Reply:
https://carbon-design-system.github.io/carbon-charts/?path=/story/tutorials--tabular-data-format

Reply:
Added a `ChartTabularData` type to the core model interfaces, which is pretty loose: `Record<string, any>[]`  

Haven't yet removed mentions of `datasets/labels` etc..